docs/prestodb-guide.md

```diff
@@ -238,10 +238,10 @@ Now you can use the Presto CLI on the coordinator to query data sources in the c
   ```
   Replace the spark and hadoop version with the version used in your cluster.
   For example, use prestodb profile and 
-  if you are using Spark 2.4.4, you would like to compile using:
```

Comment:
prestosql-guide also has this.

please look up 2.4.4

Reply:
fixed